internal/pkg/service/common/etcdop/op/util.go

```diff
@@ -0,0 +1,55 @@
+package op
+
+import (
+	"bytes"
+	"sort"
+)
+
+func removeOpsOverlaps(ops []TrackedOp) (out []TrackedOp) {
+	// Order operations with the smallest key and with the largest range to begin
+	sort.SliceStable(ops, func(i, j int) bool {
```

Comment:
Zoradime si to tak:
- podla `key`, nizsi skor
- podla dlzky, ak je `key` rovnaky